css/screens/landingPage.css

```diff
@@ -0,0 +1,136 @@
+/*  ==== css for landing page === */
+
+.big-wrapper{
+    padding: 1.7rem 0 2rem;
+    width: 100%;
+    min-height: 100vh;
+    overflow: hidden;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+.container{
+    max-width: 81rem;
+    width: 100%;
+    margin: 0 auto;
+    padding: 0 3rem;
+}
+header .container{
+    position: relative;
+    z-index: 10;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.logo{
+    display: flex;
+    align-items: center;
+}
+.logo>h3{
+    font-size: var(--h3-font-size);
+    font-weight: var(--font-semi-bold);
+}
+.logo img{
+    width: 50px;
+    margin-right: .6rem;
+}
+.links >ul{
+    display: flex;
+    align-items: center;
+}
+.links a{
+    color: var(--text-color) ;
+    margin-left: 4.5rem;
+    display: inline-block;
+    transition: .1s;
+}
+.links a:hover{
+    color: var(--primary-dark);
+    transform: scale(1.05);
+}
+.btn{
```

Comment:
Generally `btn` in global class. So you can either move it to the utility folder or can change the name to more specific towards the section.

Reply:
sure